src/util/voxelutility.cpp

```diff
@@ -6,8 +6,7 @@
 #include <string>
 #include <vector>
 
-// TODO(nino): lowercase this
-namespace VoxelUtility {
+namespace voxel_utility {
```

Comment:
A better name would be `io_utils` with functions `load_from_qb` and `save_to_qb`

Reply:
ya, I'll put the mesh generators here.